### ISSUE TITLE
[$15][MSFT-149] [Challenge Submission Page] Won't focus on 'I Understand and Agree' when user use the tap key

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2840]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges/30095032/submit
2. Press tap key to go to 'I Understand and Agree'

**Expected Result**
Should be able to select that option using tab key

**Actual Result**
Won't focus on 'I Understand and Agree' when user use the tap key

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.1.1 Keyboard
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F56

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096182 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096182 has been updated - it has been assigned to anonymousjaggu.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2840 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2840 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2840 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2840 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2840 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2840 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097373 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097373 has been updated - it has been assigned to anonymousjaggu.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097373 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
@veshu ticket prize yet to be doubled. Thanks!

**Comment by drasticdpk** [comment]
Operating System: Windows
Browser: Chrome
URL : https://test-community-app.topcoder-dev.com/challenges/30055274/submit

**Comment by drasticdpk** [comment]
Verified on
Operating System: Windows
Browser: Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097373<br/><br/>```This is an automated message for crazyk via Topcoder X```
